README.md

```diff
@@ -17,6 +17,8 @@ Each plugin may have it's own build instructions in the linked README.md.
 
 * [VFIO](docs/vfio/README.md)
 * [KVM](docs/kvm/README.md)
+* Network
```

Comment:
What about just calling the plugin network-bridge? How many more plugins do you think we could have for network/ package?

Reply:
bridge, ipv6, and maybe an overlay type - that#s about the scope I'd like to play with.

Reply:
In pkg/ it makes more sense to keep it structured like that. There will be some shared network modules.

I'd like to keep it symmetric -- the same dir structure in pkg, doc and cmd.